jekyll/_cci2_ja/server-3-install-build-services.adoc

```diff
@@ -732,29 +732,28 @@ NOTE: 静的 JSON 認証情報から Workload Identity に切り替える場合
 
 . *サーバーを設定します。*
 +
-VM サービスをKOTS 管理者コンソールから設定します。 利用可能な設定オプションの詳細については、 https://circleci.com/docs/2.0/server-3-operator-vm-service[VM サービス]ガイドを参照してください。
+VM サービスを KOTS 管理者コンソールから設定します。 利用可能な設定オプションの詳細については、 https://circleci.com/docs/2.0/server-3-operator-vm-service[VM サービス]ガイドを参照してください。
 +
 フィールドの設定が完了したら、*設定を保存し*、更新したアプリケーションをデプロイします。
 
 endif::env-aws[]
 
 ==== VM サービスの検証
 
-CircleCI Server の設定が完了したら、VM サービスが適切に動作しているか確認する必要がありあます。 CircleCI Server 内で、realitycheckプロジェクトを再実行できます。 緑色であれば VM サービスジョブは完了してます。 この時点で、すべてのテストが緑色で合格しているはずです。
+CircleCI Server の設定とデプロイが完了したら、VM サービスが適切に動作しているか確認する必要がありあます。 CircleCI Server 内で、realitychecker プロジェクトを再実行できます。 VM サービスジョブは完了しているはずです。 この時点で、すべてのテストが合格しているはずです。
```

Comment:
the project name is realitycheck, as per the original text 🤓 
https://github.com/circleci/realitycheck

```suggestion
CircleCI Server の設定とデプロイが完了したら、VM サービスが適切に動作しているか確認する必要がありあます。 CircleCI Server 内で、realitycheck プロジェクトを再実行できます。 VM サービスジョブは完了しているはずです。 この時点で、すべてのテストが合格しているはずです。
```